test/hasOwn_test.js

```diff
@@ -0,0 +1,60 @@
+goog.module('datalayerhelper.plain.testing.hasOwn');
+goog.setTestOnly();
+
+const {hasOwn} = goog.require('plain');
+
+describe('the isPlainObject method', function() {
+  /**
+   * Ensure that plain.hasOwn Determines if the value has a
+   * non-inherited property with the given key.
+   * @param {*} value The value to check.
+   * @param {string} key The property name.
+   * @param {boolean} expected If the value has a property named by key.
+   */
+  function assertHasOwn(value, key, expected) {
+    expect(hasOwn(value, key)).toBe(expected);
+  }
+
+  it('Assert that hasOwn recognizes simple type to not have properties', () => {
```

Comment:
I think you remove all of the "Assert that hasOwn". When the test runs, it will prefix each phrase with your describe statement. So if your describe above is "The hasOwn function", then (after my suggestion), the test would read:

"The hasOwn function recognizes simple type to not have own properties".